modules/account-lib/test/unit/coin/stx/transactionBuilder/transferBuilder.ts

```diff
@@ -202,6 +203,62 @@ describe('Stx Transfer Builder', () => {
       should.deepEqual(txJson.fee.toString(), '180');
     });
 
+    it('get pubkey of a transfer transaction signed 1', async () => {
+      const txBuilder2: any = factory.getTransferBuilder();
+      txBuilder2.from(testData.SIGNED_TRANSACTION_PK_1_3);
+      should.deepEqual(txBuilder2._fromPubKeys, [testData.pub1, testData.pub2, testData.pub3]);
+    });
+
+    it('get pubkey of a transfer transaction signed 2', async () => {
+      const txBuilder2: any = factory.getTransferBuilder();
+      txBuilder2.from(testData.SIGNED_TRANSACTION_PK_2_3);
+      should.deepEqual(txBuilder2._fromPubKeys, [testData.pub1, testData.pub2, testData.pub3]);
+    });
+
+    it('a transfer transaction signed multiple times with mid key no signer', async () => {
+      const builder = initTxBuilder();
+      builder.memo('test');
+      builder.sign({ key: testData.prv1 });
+      builder.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
+      builder.numberSignatures(2);
+      const tx = await builder.build();
+      const txBuilder2 = factory.getTransferBuilder();
+      txBuilder2.from(tx.toBroadcastFormat());
+      txBuilder2.sign({ key: testData.prv3 });
+      const signedTx = await txBuilder2.build();
+
+      const txJson = signedTx.toJson();
+      should.deepEqual(signedTx.toBroadcastFormat(), testData.SIGNED_TRANSACTION_PK_1_3);
+
+      should.deepEqual(signedTx.signature.length, 2);
+      should.deepEqual(txJson.fee.toString(), '180');
+      should.deepEqual(txJson.payload.to, testData.TX_RECIEVER.address);
+      should.deepEqual(txJson.payload.memo, padMemo('test'));
+      should.deepEqual(txJson.payload.amount, '1000');
+    });
+
+    it('a transfer transaction signed multiple times with first key no signer', async () => {
+      const builder = initTxBuilder();
+      builder.memo('test');
+      builder.sign({ key: testData.prv2 });
+      builder.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
+      builder.numberSignatures(2);
+      const tx = await builder.build();
+      const txBuilder2 = factory.getTransferBuilder();
+      txBuilder2.from(tx.toBroadcastFormat());
+      txBuilder2.sign({ key: testData.prv3 });
+      const signedTx = await txBuilder2.build();
+
+      const txJson = signedTx.toJson();
+      should.deepEqual(signedTx.toBroadcastFormat(), testData.SIGNED_TRANSACTION_PK_2_3);
```

Comment:
can we also verify that these broadcastablt tx's are not going to be rejected by the node? In short, can you make sure the only issue is `insufficient balance` from the sender when you broadcast it to the node, or better yet, can you fund the sender's address on testnet then try to broadcast this signed tx to make sure it's valid?

Reply:
Same goes for these other tx's

Reply:
The raw tx was built in coin-sandbox. Actually, this Tx does not have enough fee to work. But you can try creating a similar one that works. Look at the following reference.

https://github.com/BitGo/coins-sandbox/pull/85/files#diff-cd398d1a67e4543c01e6ec623432f224e55834573abf74cbc11eda29a17cc50bR18

Reply:
https://github.com/BitGo/coins-sandbox/blob/8cd53d9c0d69d53b72434a86019ea86a1cae34a1/stack/sendMultiSigMultiStep.js#L18